index.html

```diff
@@ -173,6 +173,44 @@ <h2>Calls for implementation and feedback</h2>
+
+A constraint is a restriction on the proposal, and may be used to withhold consensus for advancement. It takes the form of a well stated reason for why a proposal should not advance in it's present form, that is recorded by the committee. Well stated reasons should ideally illustrate the core issue of the constraint, this may be expressing a member industry position (such as a given design not working in the embedded space), a design concern, or a technical concern, and should provide an actionable plan whenever possible. The committee does not have an established concept of a rejected proposal.
+
+A block is the same as a constraint, however it is presented in a non-actionable way, or it cannot be addressed in the current stage and requires significant rework. This requires the champion to rework the proposal at an earlier stage in order to address the concern.
+
+In all possible cases, withholding consensus should take the form of a constraint. Blocks happen but should be avoided.
+
+<p>When a champion proposes a feature for advancement, a delegate may pose additional constraints and ask that the champion incorporate them into the proposal. The delegate should expect to spend time working with the champion to help flesh out the constraint and learn about other constraints that went into the proposal.
+
+<p>For features advancing from Stage 1 to Stage 2, a delegate may pose a constraint as necessary for advancement to Stage 3. In this situation, the delegate should expect to work with the champion and other delegates during Stage 2 to incorporate their constraint into the solution. In general, this is a good time to pose new constraints.
+
+<p>For features advancing to Stage 3, a delegate may ask that the champion work with them to incorporate a new constraint as a condition of advancement to Stage 3. In particular, given that consensus on Stage 3 means "the solution is complete", i.e., all open design issues havee beeen resolved including anticipated implementation and ecosystem compatibility issues, all TC39 participants should validate the design of proposals they care about before granting Stage 3 consensus.
+
+<p>Delegates should openly give feedback on proposals, especially when they have significant concerns, and especially for a proposal for stage advancement where the concern is relevant to the stage. Even if a Delegate does not intend to withhold consensus they should raise their concerns for general discussion.
+
+<p>A delegate may also request additional time to consider the proposal.
+
+<p>Conditional advancement is time-limited, giving the person raising the concern time to discuss with the champions and authors about their concerns. If a proposal has a conditional advancement, an issue must be opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.
```

Comment:
Let's step back and say what conditional advancement is. I think we usually do this more in cases where it's a minor editorial issue to resolve, and we already have the solution in mind.

Also, this paragraph seems a bit out of nowhere--should we shift it into its own heading?

```suggestion
<p>The committee may resolve to <em>conditionally advance</em> a proposal to resolve a particular well-understood condition offline, e.g., making a particular small specification change concrete, among a group of interested people who have an idea of the solution. Conditional advancement is time-limited, giving the person raising the concern time to discuss with the champions and authors about their concerns. If a proposal has a conditional advancement, an issue must be opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.
```

Reply:
sounds good to me. I will do that in my next pass.